config/config.go

```diff
@@ -35,6 +36,9 @@ func (c *Config) applyDefaults() {
 	if c.RepoPath == "" {
 		c.RepoPath = ".repos"
 	}
+  if c.ReadmePath == "" {
+    c.ReadmPath = "README.md"
```

Comment:
Hi @bolshoytoster this looks like a typo that's causing it to fail to build. Should be `c.ReadmePath` not `c.ReadmPath`.

Reply:
No worries! I think you may have rebased a bit too far back as this includes some earlier changes and skips a commit. Can you do a fetch and rebase your changes on top of 2a1313d? There's also a missing `_`, I'll leave a comment in PR where it's at.